enioka_scan/src/main/java/com/enioka/scanner/sdk/zebraoss/ssi/SsiMultiPacket.java

```diff
@@ -0,0 +1,82 @@
+package com.enioka.scanner.sdk.zebraoss.ssi;
+
+import android.util.Log;
+
+import com.enioka.scanner.data.Barcode;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * SSI multi-packet, containing the header and checksum bytes. Suitable for both incoming.
+ * So far, only DECODE_DATA packets seem to sometimes use multi-packets for large barcodes, so this
+ * class is only used as a buffer for reading those types of packets before being converted to a
+ * barcode with no other use. For this reason, many sanity checks are skipped.
+ */
+public final class SsiMultiPacket {
```

Comment:
no test it seems for multi packet, please add one from your captures

Reply:
done